frontend/web/App.css

```diff
@@ -0,0 +1,16 @@
+.app-container {
+    padding: 20px;
```

Comment:
Generic class that's unused except for this pr, can be removed and just replaced with className=p-4